### PR TITLE
[zh] sync storage-classes.md & volumes.md in storage

### PR DESCRIPTION
**Pull request by howieyuen**
related to https://github.com/kubernetes/website/issues/29325
task 10

### CODE REVIEW

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -316,6 +316,43 @@ to see its supported topology keys and examples.
...
 但是你需要查看特定 CSI 驱动程序的文档以查看其支持的拓扑键名和例子。
 
+{{< note >}}
+<!-- 
+   If you choose to use `WaitForFirstConsumer`, do not use `nodeName` in the Pod spec
+   to specify node affinity. If `nodeName` is used in this case, the scheduler will be bypassed and PVC will remain in `pending` state.
+
+   Instead, you can use node selector for hostname in this case as shown below.
+-->
+   如果你选择使用 `WaitForFirstConsumer`，请不要在 Pod 规越中使用 `nodeName` 来指定节点亲和。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:326]:
```suggestion
   如果你选择使用 `WaitForFirstConsumer`，请不要在 Pod 规约中使用 `nodeName` 来指定节点亲和性。
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 2]
```diff
@@ -316,6 +316,43 @@ to see its supported topology keys and examples.
...
+   如果在这种情况下使用 `nodeName`，将会绕过调度程序，PVC 将停留在 `pending` 状态。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:327]:
```suggestion
   如果在这种情况下使用 `nodeName`，Pod 将会绕过调度程序，PVC 将停留在 `pending` 状态。
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 3]
```diff
@@ -647,6 +684,41 @@ parameters:
...
+  path: /share
+  readOnly: false
+```
+
+<!-- 
+* `server`: Server is the hostname or IP address of the NFS server.
+* `path`: Path that is exported by the NFS server.
+* `readOnly`: A flag indicating whether the storage will be mounted as read only (default false).
+-->
+* `server`：服务器是 NFS 服务器的主机名或 IP 地址。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:706]:
```suggestion
* `server`：NFS 服务器的主机名或 IP 地址。
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 4]
```diff
@@ -647,6 +684,41 @@ parameters:
...
+* `path`：NFS 服务器导出的路径。
+* `readOnly`：是否将存储挂载为只读的标志（默认为 false）。
+
+<!-- 
+Kubernetes doesn't include an internal NFS provisioner. You need to use an external provisioner to create a StorageClass for NFS.
+Here are some examples:
+* [NFS Ganesha server and external provisioner](https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner)
+* [NFS subdir external provisioner](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner)
+-->
+Kubernetes 不包含内部 NFS 供应商。你需要使用外部供应商为 NFS 创建 StorageClass。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:716]:
```suggestion
Kubernetes 不包含内部 NFS 驱动。你需要使用外部驱动为 NFS 创建 StorageClass。
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 5]
```diff
@@ -647,6 +684,41 @@ parameters:
...
+这里有些例子：
+* [NFS Ganesha 服务器和外部供应商](https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner)
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:718]:
```suggestion
* [NFS Ganesha 服务器和外部驱动](https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner)
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 6]
```diff
@@ -647,6 +684,41 @@ parameters:
...
+* [NFS subdir 外部供应商](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner)
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:719]:
```suggestion
* [NFS subdir 外部驱动](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner)
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 7]
```diff
@@ -912,6 +984,17 @@ parameters:
...
+{{< feature-state for_k8s_version="v1.22" state="deprecated" >}}
+
+<!-- 
+The Quobyte in-tree storage plugin is deprecated, an 
+[example](https://github.com/quobyte/quobyte-csi/blob/master/example/StorageClass.yaml)
+`StorageClass` for the out-of-tree Quobyte plugin can be found at the Quobyte CSI repository.
+-->
+Quobyte 树内（in-tree）存储插件已弃用，
+[示例](https://github.com/quobyte/quobyte-csi/blob/master/example/StorageClass.yaml)
+可以在 Quobyte CSI 存储库中找到用于树外（in-tree）Quobyte 插件的 `StorageClass`。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:996]:
```suggestion
你可以在 Quobyte CSI 仓库中找到用于树外（in-tree）Quobyte 插件的 `StorageClass`
[示例](https://github.com/quobyte/quobyte-csi/blob/master/example/StorageClass.yaml)。
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 8]
```diff
@@ -219,23 +219,22 @@ beta features must be enabled.
 
 <!--
 To disable the `awsElasticBlockStore` storage plugin from being loaded by the controller manager
-and the kubelet, set the `CSIMigrationAWSComplete` flag to `true`. This feature requires the `ebs.csi.aws.com` Container Storage Interface (CSI) driver installed on all worker nodes.
+and the kubelet, set the `InTreePluginAWSUnregister` flag to `true`.
 -->
-如欲禁止 `awsElasticBlockStore` 存储插件被控制器管理器和 kubelet
-组件加载，可将 `CSIMigrationAWSComplete` 特性门控设置为 `true`。此特性要求在
-集群中所有工作节点上安装 `ebs.csi.aws.com` 容器存储接口驱动。
+要禁止控制器管理器和 kubelet 加载 `gcePersistentDisk` 存储插件，
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:224]: 这个到底是 gcePersistentDisk 还是 awsElasticBlockStore ？

File: content/zh/docs/concepts/storage/volumes.md [hunk 9]
```diff
@@ -589,15 +588,15 @@ targetWWNs expect that those WWNs are from multi-path connections.
...
 这样 Kubernetes 主机才可以访问它们。
-{{< /caution >}}
+{{< /note >}}
 
 <!--
-See the [FC example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/fibre_channel) for more details.
+See the [fibre channel example](https://github.com/kubernetes/examples/tree/master/staging/volumes/fibre_channel) for more details.
 -->
-更多详情请参考 [FC 示例](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/fibre_channel)。
+更多详情请参考 [光纤通道 示例](https://github.com/kubernetes/examples/tree/master/staging/volumes/fibre_channel)。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:599]:
```suggestion
更多详情请参考 [FC 示例](https://github.com/kubernetes/examples/tree/master/staging/volumes/fibre_channel)。
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 10]
```diff
@@ -883,12 +898,28 @@ You must have your own GlusterFS installation running before you can use it.
...
+HostPaths when possible. When a HostPath volume must be used, it should be scoped to only the
+required file or directory, and mounted as ReadOnly.
+
+If restricting HostPath access to specific directories through AdmissionPolicy, `volumeMounts` MUST
+be required to use `readOnly` mounts for the policy to be effective.
+-->
+HostPath 卷存在许多安全风险，最佳做法是尽可能避免使用 HostPath。
+当必须使用 HostPath 卷时，它的范围应仅限于所需的文件或目录，并以只读方式挂载。
+
+如果通过 AdmissionPolicy 限制对特定目录的 HostPath 访问，
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:919]:
```suggestion
如果通过 AdmissionPolicy 限制 HostPath 对特定目录的访问，
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 11]
```diff
@@ -956,6 +990,8 @@ Watch out when using this type of volume, because:
 -->
 当使用这种类型的卷时要小心，因为：
 
+* HostPaths 可以公开特权系统凭据（例如 Kubelet）或特权 API（例如容器运行时套接字），
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:993]:
```suggestion
* HostPath 卷可能会暴露特权系统凭据（例如 Kubelet）或特权 API（例如容器运行时套接字），
```

File: content/zh/docs/concepts/storage/volumes.md (hunk 8 above)
Comment (howieyuen) [reply content/zh/docs/concepts/storage/volumes.md:224]: 重新检查确定是 awsElasticBlockStore

**Pull request merged**